test/e2e/support/proxy.js

```diff
@@ -8,7 +8,7 @@ module.exports = class Proxy {
     this.proxyPathRegExp = null
 
     this.proxy = httpProxy.createProxyServer({
-      target: 'http://localhost:9876'
```

Comment:
This is a test code, which mimics a third-party standalone proxy application, that's why we can avoid configuring a DNS lookup, but instead skip the DNS resolution completly by pointing to the IPv4 address.

Reply:
LGTM